neofetch

```diff
@@ -965,6 +965,12 @@ get_distro() {
                         distro="Proxmox VE ${distro%/*}"
                 esac
 
+            elif [[ $DESKTOP_SESSION == regolith ]]; then
+                case $distro_shorthand in
+                    on|tiny) distro="Regolith";;
+                    *) distro="Regolith ($(lsb_release -sic))"
+                esac
+
```

Comment:
There's no `os-release`?

Reply:
There is. But the distro is not seen from that file.

Reply:
i.e. not always seen (e.g. when installing on top of ubuntu)